pkg/cmd/application/create/app_create.go

```diff
@@ -58,10 +61,15 @@ NOTE:
 				opts.LocalPath = "./" + opts.Name
 			}
 			opts.Streams = userio.NewIOStreamsWithInteractive(
-				cmd.InOrStdin(),
-				cmd.OutOrStdout(),
+				os.Stdin,
+				os.Stdout,
 				!opts.NonInteractive,
 			)
+
+			opts.Streams.Wizard(
```

Comment:
Creation of a wizard is here, but closing is in `run`. Can we move the creation also to `run`?